run_unittests.py

```diff
@@ -2163,6 +2163,33 @@ def test_command_line(self):
         self.assertEqual(obj.compiler_options['c_args'].value, ['foo bar', 'one', 'two'])
         self.wipe()
 
```

Comment:
Please edit the commit message to say "Closes #3456" so it is closed automatically.